docs/src/52-mbtiles-stats.md

```diff
@@ -0,0 +1,20 @@
+# Get a tile statistics from MBTiles file
+
+For the concern of efficiency, you could figure out the page size and file size, tile size and  bounds(represented as WGS 84 latitude and longitude values) of covered area with `mbtiles stats` command.
+
+```shell
+mbtiles stats /path/to/my_file.mbtiles
+File: ./tests/fixtures/mbtiles/world_cities.mbtiles
+FileSize: 48.00KiB
+Schema: flat
+Page size: 4.00KiB
+|  zoom   |  count  |smallest | largest | average |  bbox   |
```

Comment:
has this example changed with the new output changes?

Reply:
Updated now.